docker-compose.override.os.yml

```diff
@@ -0,0 +1,5 @@
+version: '2.1'
+services:
+    mender-gui:
+        volumes:
+            - ../gui/dist:/var/www/mender-gui/dist
```

Comment:
why is that needed? probably should be a separate commit?

Reply:
oh sorry, this is a local file which wasn't meant to be added. I'm removing it.